static/script.js

```diff
@@ -1,4 +1,14 @@
+document.querySelectorAll(".emoji-button").forEach(function (button) {
+  button.addEventListener("webkitAnimationEnd", function() {
+    // reset animation
+    button.style.animationName = "";
+  });
+});
+
 function post_feedback(fb) {
+  var button = document.querySelector(".emoji-button." + fb);
```

Comment:
This kind of slipped through in my previous review.

Why wouldn't you use `const` instead of `var` here? It seems to work on my machine (Chromium 58).